### ISSUE TITLE
NaN Particles

### ISSUE THREAD

**Issue by mauro3** [baggepinnen/MonteCarloMeasurements.jl#91]
This seems inconsistent:
```
julia> NaN ± NaN
ERROR: ArgumentError: Normal: the condition σ >= zero(σ) is not satisfied.

julia> NaN ± Inf
Particles{Float64,2000}
 NaN ± NaN
```

**Comment by baggepinnen** [comment]
It comes from this
```julia
julia> std(rand(Normal(0,Inf), 10))
NaN

julia> Normal(0,NaN)
ERROR: ArgumentError: Normal: the condition σ >= zero(σ) is not satisfied.
```

**Comment by mauro3** [comment]
It's feels a bit "wat?", but it's ok to close this.

**Comment by baggepinnen** [comment]
Do you have an application where the semantics for infinite ra dom numbers is important? If not we can probably close this issue as I'm not sure if it makes sense to modify the behavior to give something other than nan.

**Comment by mauro3** [comment]
No, my worry was more that `NaN \pm NaN` errors.  I got data at times which contains NaNs (arguably I should use `missing`) and that is how I came across this.  But  I can work around it.  Well, it might be worth having a coherent story for 1.0.
